exodus/exodus/settings/dev.py

```diff
@@ -38,9 +38,12 @@
 CELERY_BROKER_URL = 'amqp://guest@localhost//'
 BROKER_URL = CELERY_BROKER_URL
 MINIO_STORAGE_ENDPOINT = '127.0.0.1:9000'
-MINIO_STORAGE_ACCESS_KEY = 'exodusexodus'
-MINIO_STORAGE_SECRET_KEY = 'exodusexodus'
+MINIO_STORAGE_ACCESS_KEY = 'AKIAIOSFODNN7EXAMPLE'
+MINIO_STORAGE_SECRET_KEY = 'wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY'
 
 ALLOW_APK_UPLOAD = True
 
 TRACKERS_AUTO_UPDATE = True
+
+# Customization
+INSTALLED_APPS = ['men_customization'] + INSTALLED_APPS
```

Comment:
This makes every development instance be a "MEN" instance which I guess shouldn't be the case

Reply:
I'll remove this line.